CHANGELOG.md

```diff
@@ -5,6 +5,13 @@
 * [#436](https://github.com/stripe/stripe-php/pull/436) Introduce new `Idempotency` error class
 
 ## 6.0.0 - 2018-02-07
+Major version release. List of backwards incompatible changes to watch out for:
++ The minimum PHP version is now 5.4.0. If you're using PHP 5.3 or older, consider upgrading to a more recent version.
+* `\Stripe\AttachedObject` no longer exists. Attributes that used to be instanced as `\Stripe\AttachedObject` (such as `metadata`) are now instanced as `\Stripe\StripeObject`.
++ Attributes that used to instanced as PHP arrays (such as `legal_entity->additional_owners` on `\Stripe\Account` instances) are now instanced as `\Stripe\StripeObject`, except when they are empty. `\Stripe\StripeObject` has array semantics so this should not be an issue unless you are actively checking types.
```

Comment:
And I might do the same here. There's also a missing "be" in "Attributes that used to be instanced".